data/coalition/coalition outfits.txt

```diff
@@ -275,3 +298,9 @@ outfit "Coalition License"
 	category "Special"
 	thumbnail "outfit/coalition license"
 	description "Completed a lot of Coalition jobs and received permission to purchase civilian technology and ships."
+
+outfit "Heliarch License"
+	category "Special"
+	thumbnail "outfit/heliarch license"
+	description "Assisted the Heliarch in several instances, and as such was awarded the rank of Heliarch. This circlet was originally designed for a Saryd, but was modified to better fit a human head."
```

Comment:
their description is `And the third species are centaurs. Not just something vaguely resembling centaurs; they look like they could have stepped right off the page of a story book from ancient Earth.` , while true that as such the upper body should be exactly that of a human, "right off the page of a story book" rather than "right off of Greek mythology" leaves space for them being centaurs as are described in fiction which may not depict them as being literally the Greek centaurs. I personally prefer suggesting that there *are* physical differences between the upper body of a Saryd and that of a human, even if it's just size or some details that would affect how a circlet is made, because a perfectly human upper body seems a bit immersion-breaking to me